content/blog/r-cpp-testing.md

```diff
@@ -0,0 +1,88 @@
+---
+author: "Giovanni Charles"
+date: 2020-03-27
+title: "Gotta catch 'em all"
+best: false
+tags:
+ - R
+ - C++
+---
+
+Great packages have great test suites. Packages with C++ are not exceptions. You
+might be surprised to find how easy it is to add C++ unittests to your package.
+
+This post will focus on [testthat](https://github.com/r-lib/testthat)'s
+[Catch](https://github.com/catchorg/Catch2) integration.
+There are plenty of C++ testing frameworks, each with their own pros and cons.
```

Comment:
I think that there might be something missing here in the outline of the problem - namely, if you've written tests in your preferred C++ framework, how do you arrange to run it from R and integrate it with its QA tools

Reply:
That would be cool! But I would be talking from outside my experience.

Perhaps I should remove this sentence so that it doesn't lead people on?